firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -3700,6 +3700,130 @@ extension String {
         comment: "Question shown to user when tapping an SMS or MailTo link that opens the external app for those.")
 }
 
+// MARK: Enhanced Tracking Protection/Unified Trust Panel
+extension String {
+    public static let EnhancedTrackingProtectionOnTitle = MZLocalizedString(
+        key: "Menu.EnhancedTrackingProtection.On.Title",
+        tableName: nil,
```

Comment:
Can we create a struct `EnhancedTrackingProtection` and specify it as the `tableName`? Seems like we want to specify `tableName` going forward.

Reference: https://github.com/mozilla-mobile/firefox-ios/wiki/Strings,-%60AccessibilityIdentifiers%60,-%60ImageIdentifiers%60#strings

Reply:
awesome, thank you!